europy/helpers.py

```diff
@@ -0,0 +1,19 @@
+import os, yaml, json
+from europy.lifecycle import reporting
+
+def load_global_params(path, report=True):
```

Comment:
is this used anywhere?

Reply:
in the notebook example I'm working on, will commit it up soon in examples. This is another things we can talk about., but just trying to get something end-to-end